apps/bfd-pipeline/bfd-pipeline-rif-load/src/main/java/gov/cms/bfd/pipeline/rif/load/RifLoader.java

```diff
@@ -614,6 +588,86 @@ private void processAsync(
     }
   }
 
+  /**
+   * Applies various "tweaks" to the {@link Beneficiary} (if any) in the specified {@link
+   * RifRecordEvent}:
+   *
+   * <ul>
+   *   <li>Hashes any MBIs or HICNs in it.
+   *   <li>Updates its {@link Beneficiary#getBeneficiaryMonthlys()} records, as needed.
+   *   <li>Adds a {@link BeneficiaryHistory} record for previous the {@link Beneficiary}, as needed.
+   * </ul>
+   *
+   * @param entityManager the {@link EntityManager} to use
+   * @param loadedBatchBuilder the {@link LoadedBatchBuilder} to use
+   * @param rifRecordEvent the {@link RifRecordEvent} to handle the {@link Beneficiary} (if any) for
+   */
+  private void tweakIfBeneficiary(
```

Comment:
moving this work into it's own method will make this much easier to maintain. thx.